monero-outreach-docs/translations/es/storyboard/story58.md

```diff
@@ -0,0 +1,67 @@
+bien intencionados perdieran su dinero. [Estafas crecieron desenfrenadamente](https://research.bloomberg.com/pub/res/d28giW28tf6G7T_Wr77aU0gDgFQ) bajo auspicios de la comercialización de monedas promocionadas para resolver aparentemente todos los problemas del mundo. 
+Ahora, como reguladores alrededor del mundo están apretando fuertemente en estas estafas, nos encontramos en un punto bajo de confianza en criptomonedas como un todo. Muchos inversionistas han agrupado
+todas las criptomonedas como estafas después de las graves ofensas del 2017, a pesar de que algunos proyectos de monedas se mantuvieron fuera del alboroto.
+
+Monero es uno de ellos. El Proyecto Monero (XMR) ha mantenido un propósito en mente: transferencias de valor rápidas, de bajo costo y anónimas. Monero nunca depende de campañas de mercado costosas, ICOs o
+financiación de riesgo. Cuenta con una comunidad basada en código abierto de desarrolladores, usuarios, creativos y luchadores por la libertad. Nosotros la comunidad no creemos que Monero, o criptomoneda para esa causa,
+puede solucionar todos los problemas del mundo. Nosotros creemos que Monero puede solucionar un problema importante: transferir dinero a cualquiera de manera privada y sin autoridad central.
+
+## Buena Tecnología vs. Buen Marketing
+
+Monero ha estado implementando tranquilamente un estado del arte tecnológico y matemáticas para mejorar la privacidad. El equipo de ingeniería no es segundo a nadie. Los desarrolladores trabajan más cohesivamente que
+muchas universidades de investigación y [corporaciones de Fortune 500](https://www.nytimes.com/2015/08/16/technology/inside-amazon-wrestling-big-ideas-in-a-bruising-workplace.html), sin embargo son financiados por la comunidad, no grandes negocios.
+
+La comunidad trabaja incansablemente para apoyar a los recién llegados a través de IRC, Telegram y foros como StackExchange. Todos intentan contribuir con el conocimiento técnico 
+con el que cuentan. No hay equipos de apoyo de pagos o servicio al cliente. Monero Monero tiene una comunidad en Telegram de 9k+ miembros que está activa 365 días y recibe cualquier pregunta, sin importar
+tu conocimiento técnico.
+
+Con un fuerte equipo de ingenieros, una grande comunidad y una historia implementando gran tecnología, Monero se mantiene aparte de monedas que dependen principalmente en tácticas de marketing.
+Nuevos proyectos son primero mostrados en el [foro de la comunidad](https://forum.getmonero.org/). Cualquiera puede comentar, votar y prometer sus propias monedas Monero para financiar sus proyectos favoritos. Una vez que un proyecto está en marcha, cualquiera puede ver los
+hitos y productos de trabajo. Esto mantiene al equipo del proyecto responsable y permite a los patrocinadores ver los resultados de los fondos del proyecto. Puedes ver exactamente cómo 

```

Comment:
_Hito_ is not very common these days since it's an older word but it's correct inside the context since it means "key fact", rather than a synonym like _evento_ (event), _hecho_ (fact) or _acontecimiento_ (occurrence), in my opinion. What do you think?

And work products is just that, a product resulting from the work of the project.

Reply:
I still think "hitos" is not a common word, is not used un most of the countries, so we should find a word that fits a bigger range of regions. We can't just follow our own thoughts.

Reply:
Well then, what do you think about this:

_... cualquiera puede ver los eventos clave y productos de trabajo._

Reply:
I believe milestones mean, "stages" of the project, not the "events" of a project. Events for me make more reference to "special" events as if you were going to an event.

Reply:
A definition looking at Google is: "an action or event marking a significant change or stage in development", therefore, an important event in the development of a project. OK you also have a point since event can be taken also as a stage but since we have "stage", a synonym, _etapa_ is also a synonym.